Grammar Update to organizations.md (#5605)
umbraco/UmbracoDocs: umbraco-cloud/getting-started/the-umbraco-cloud-portal/organizations.md

# Organizations

> On Umbraco Cloud it is possible to setup an Organization. An organization is handy if you need to manage permissions for multiple people (such as developers to content editors) or are managing a lot of projects for different customers.

Reviewer: ```suggestion
On Umbraco Cloud it is possible to setup an Organization. An organization is handy if you are managing many projects for different customers. It is also handy if you need to manage permissions for multiple users (such as developers, content editors etc.). 
```

> With an organization, you get an overview of all projects and members that are part of it, manage payment methods for projects, as well as many other functions outlined on this page.

Reviewer: ```suggestion
With an organization, you get an overview of all projects and members that are part of it. You can also manage payment methods for projects, as well as many other functions outlined on this page.
```

### Information

> In the **Information** section of the Organization, you can find all your details about your Organization. If there are any changes to your detais, you can change them here.

Reviewer: ```suggestion
In the **Information** section of the Organization, you can find all the details about your Organization. If there are any changes to your details, you can change them here.
```

![Information](images/Information-v10.png)

### Members

> In the **Members** section, you can view current members, view pending invites, and see the Multi-Factor Authentication (MFA) status for the Members of your Organization. You can also set up different permissions for your Members, such as Read, Write, and Administrators for your organization by adjusting their **Role**.

Reviewer: ```suggestion
In the **Members** section, you can view current members, pending invites, and see the Multi-Factor Authentication (MFA) status for the Members of your Organization. You can also setup different permissions for your Members, such as Read, Write, and Administrators for your organization by adjusting their **Role**.
```

* See Organization Members

{% hint style="info" %}
Being a Member of an organization, does not give access to any projects under it. To get access to a project under an organization, you need to be [invited](/umbraco-cloud/set-up/users-on-cloud.md) to the project. This can be done by either someone that is already part of the project or an administrator in your organization.
{% endhint %}

### Multi Factor Authentication (MFA) enforcement

> When working in organizations on Umbraco Cloud, as a company, you might want to enforce a certain type of MFA method for members.

Reviewer: ```suggestion
When working in organizations on Umbraco Cloud, as a company, you might want to enforce a certain type of Multi-Factor Authentication (MFA) method for members.
```

> When working in organizations on Umbraco Cloud, as a company, you might want to enforce a certain type of Multi-Factor Authentication (MFA) method for members.

Reviewer: ```suggestion
When working in organizations on Umbraco Cloud, as a company, you can enforce a certain type of Multi-Factor Authentication (MFA) method for members.
```

4. Find the member that needs to have MFA enabled.
5. Click on the cogwheel and select the **Enforced MFA Method** from the drop-down list for the member.

> Once it has been enabled, the next time the member logs in, they will be forced to set up the chosen MFA method.

Reviewer: ```suggestion
Once it has been enabled, the next time the member logs in, they will be forced to set up the chosen Multi-Factor Authentication (MFA) method.
```

> Once it has been enabled, the next time the member logs in, they will be forced to set up the chosen Multi-Factor Authentication (MFA) method.

Reviewer: ```suggestion
Once it has been enabled, the next time the member logs in, they will be forced to setup the chosen Multi-Factor Authentication (MFA) method.
```
